Virtusize/Sources/VirtusizeRepository.swift

```diff
@@ -120,36 +120,36 @@ internal class VirtusizeRepository: NSObject {
 	///   - productId: the product ID from the Virtusize server
 	///   - onSuccess: the closure is called if the data from the server is successfully fetched
 	internal func fetchInitialData(
-		externalProductId: String,
-		productId: Int?,
-		onSuccess: (VirtusizeServerProduct
-		) -> Void) {
+        externalProductId: String,
+        productId: Int?,
+        onSuccess: @escaping (VirtusizeServerProduct) -> Void
```

Comment:
https://docs.swift.org/swift-book/documentation/the-swift-programming-language/closures/